alpine-rails-buildkite/etc/sv/buildkite

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+export HOME=/var/buildkite
+export GEM_HOME=${HOME}/.gem
+export BUILDKITE_AGENT_VERSION=edge
+export BUILDKITE_BUILD_PATH=/var/buildkite/builds
+export BUILDKITE_HOOKS_PATH=/var/buildkite/hooks
+export BUILDKITE_BOOTSTRAP_SCRIPT_PATH=/usr/lib/buildkite/bootstrap.sh
+
+exec chpst -u buildkite -- /usr/bin/buildkite-agent start
```

Comment:
I think that each user should have his dedicated group only that user is member of, instead of `nogroup`.

Grouping users that have no group into `nogroup` creates a group of those users, and so they by accident can share resources.

If a user with primary group `nogroup` creates a directory in `/tmp` and sets permissions `0750` (default umask!) they might not intend the other users with the same primary group can access the contents.

For that reason I consider a "primary group per primary user" as the best isolation, and typically default to it.